nautobot/ipam/forms.py

```diff
@@ -111,6 +111,9 @@ class VRFForm(NautobotModelForm, TenancyForm):
     namespace = DynamicModelChoiceField(queryset=Namespace.objects.all())
     devices = DynamicModelMultipleChoiceField(queryset=Device.objects.all(), required=False)
     virtual_machines = DynamicModelMultipleChoiceField(queryset=VirtualMachine.objects.all(), required=False)
+    virtual_device_contexts = DynamicModelMultipleChoiceField(
```

Comment:
Should we have add/remove equivalents to this in the bulk-edit form?

Reply:
https://github.com/nautobot/nautobot/issues/6938